gradle/scripts/remote-lib.gradle

```diff
@@ -0,0 +1,32 @@
+import java.nio.file.Paths
+
+import de.undercouch.gradle.tasks.download.DownloadAction
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath group: 'de.undercouch', name: 'gradle-download-task', version: '3.4.3'
+    }
+}
+
+def remoteLibsDir = file('.remote-libs')
+
+ext.remoteLib = { url ->
```

Comment:
Unfortunately, I had to re-introduce this functionality in order to download the JavaFX dependency from the `assets` repo.  To keep things clean, I extracted all the related code into this standalone script.

Reply:
:(((((